jumanji/environments/__init__.py

```diff
@@ -39,8 +43,10 @@ def is_colab() -> bool:
 
         if is_colab():
             backend = "inline"
-        else:
+        elif is_notebook():
```

Comment:
It seems to me that a PyCharm console has `"ipykernel" in sys.modules` True. Which means the `IPython.get_ipython().run_line_magic("matplotlib", backend)` line would still break on PyCharm console.

Reply:
Have you tested this in the PyCharm Python Console? On my end, `"ipykernel" in sys.modules` returns `False`